flax/core/nn/pooling.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2024 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
`flax/core/nn` mostly contains stuff for Linen's functional API, I'm not sure this is the best place to put `pooling`. Can you move it to `flax/pooling.py`?

Reply:
Updated according to your comment